lostandfound/urls.py

```diff
@@ -14,20 +14,9 @@
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
 from django.conf.urls import url, include
+from django.contrib import admin
 
-from rest_framework import routers
-
-from lostandfound.views import UserViewSet, GroupViewSet
-
-
-# Routers provide an easy way of automatically determining the URL conf.
-router = routers.DefaultRouter()
-router.register(r'users', UserViewSet)
-router.register(r'groups', GroupViewSet)
-
-# Wire up our API using automatic URL routing.
-# Additionally, we include login URLs for the browsable API.
+admin.site.site_header = "Lost and Found Administration"
 urlpatterns = [
-    url(r'^', include(router.urls)),
```

Comment:
since we have distinctly separated the app directory from the project, and the API lives in the app directory,  i think it should be in the app's urls.py file. I am not totally sure thou, but we can confirm from hassan.